lib/mayaUsd/render/MaterialXGenOgsXml/libraries/mx_lighting_maya_none.glsl

```diff
@@ -6,14 +6,14 @@
 // knowing that the Maya sampling functions are there, otherwise compilation will fail unless
 // there is an IBL active in the Maya lighting.
 
-#include "pbrlib/genglsl/lib/mx_microfacet_specular.glsl"
+#include "libraries/pbrlib/genglsl/lib/mx_microfacet_specular.glsl"
```

Comment:
These are the latest versions for MaterialX 1.38.4. Changes a for the "library" changes in shadergen, and roughness being replaced with alpha in the GGX implementations.